src/code42cli/cmds/departing_employee.py

```diff
@@ -25,6 +27,18 @@ def departing_employee(state):
     pass
 
 
+@departing_employee.command("list")
+@format_option
+@sdk_options()
+def _list(state, format):
+    employee_generator = _get_departing_employees(state.sdk)
+    employee_list = [
+        employee for employees in employee_generator for employee in employees["items"]
```

Comment:
I would prefer if we avoid putting multiple instances of `for` on the same line.